modules/packages/Sort.chpl

```diff
@@ -30,14 +30,26 @@ Comparators
 
 Comparators allow sorting data by a mechanism other than the
 default comparison operations between array elements. To use a comparator,
-define a record with either a ``key(a)`` or ``compare(a, b)`` method, and pass
-an instance of that record to the sort function (examples shown below).
+define a record or a class with an appropriate method and then pass
+an instance of it to the sort function. Examples are shown below.
 
-If both methods are implemented on the record passed as the comparator, the
-``key(a)`` method will take priority over the ``compare(a, b)`` method.
+Comparators need to include at least one of the following methods:
 
-Key Comparator
-~~~~~~~~~~~~~~
+ * ``key(a)``  -- see `The .key method`_
```

Comment:
I wish these could be links themselves, but then they wouldn't be monospaced. :/